### PR TITLE
fix: compatibility for some old chromium core

### PR DESCRIPTION
**Pull request by Parsifa1**
little fix for typst-preview

### CODE REVIEW

File: tools/typst-preview-frontend/src/main.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ function retrieveWsArgs() {
 
     /// The string `ws://127.0.0.1:23625` is a placeholder
     /// Also, it is the default url to connect to.
-    let url = "ws://127.0.0.1:23625";
+    let url = "ws://" + window.location.host;
```
Comment (Enter-tainer) [inline tools/typst-preview-frontend/src/main.js:40]: not sure if this is correct

Comment (Parsifa1) [reply tools/typst-preview-frontend/src/main.js:40]: maybe `window.location["host"]`?

Comment (Parsifa1) [reply tools/typst-preview-frontend/src/main.js:40]: image

works well on newest chrome

Comment (Parsifa1) [reply tools/typst-preview-frontend/src/main.js:40]: you are right，let me think...

Comment (Parsifa1) [reply tools/typst-preview-frontend/src/main.js]: fixed.
i've tested the fix.
image

File: crates/tinymist/src/tool/preview.rs [hunk 2]
```diff
@@ -631,7 +631,11 @@ pub async fn preview_main(args: PreviewCliArgs) -> anyhow::Result<()> {
 
     bind_streams(&mut previewer, websocket_rx);
 
-    let frontend_html = frontend_html(TYPST_PREVIEW_HTML, args.preview_mode, "/");
+    let frontend_html = frontend_html(
+        TYPST_PREVIEW_HTML,
+        args.preview_mode,
+        &format!("ws://{}", args.data_plane_host),
```
[the remark this answers is not in the document]
Comment (Parsifa1) [reply crates/tinymist/src/tool/preview.rs:637]: fair

[the remark this answers is not in the document]
Comment (Parsifa1) [reply crates/tinymist/src/tool/preview.rs:637]: I can't think of a better fix

**Pull request closed without merging**